Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -993,6 +993,9 @@ public Branch setPreinstallPartner(@NonNull String preInstallPartner) {
         return this;
     }
 
+     String getInstallMetaData(String key) {
```

Comment:
Theoretically... you can get PrefHelper directly without having to go through Branch to do so.

`PrefHelper.getInstance(context)`

That way you don't have to add another method to Branch.java.

Reply:
👌